src/component/EndPointCheck.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useMemo } from 'react'
+
+import { Accordion, AccordionPanel, Box, TextArea } from 'grommet'
+
+import { getGraphqlNodeEndpoint } from '../apis/nodegraphql'
+import { StatusText, StatusTitle } from './APIStatus'
+
+import * as types from '../types'
+
+function EndPointCheck() {
+    const [status, setStatus] = useState(0)
+    const [prod_env, setProdRes] = useState<types.status>({ status: 0, detail: '' })
+
+    useMemo(async () => {
+        setStatus(0)
+        let isCanceled = false
+
+        const [prod_env] = await Promise.all([
+            getGraphqlNodeEndpoint(),
+        ]);
+    
+        if (!isCanceled) {
+          setProdRes(prod_env);
+          if (prod_env.status === 1) {
+            setStatus(1)
+          } else {
+            setStatus(2)
```

Comment:
what's the purpose of `setStatus(2)`

Reply:
it would effect heading component's css setting, setStatus(2) represent error occur, you can check source code from APIStatus.tsx.

Reply:
ok